connect/json/src/test/java/org/apache/kafka/connect/json/JsonConverterTest.java

```diff
@@ -131,8 +130,8 @@ public void doubleToConnect() {
 
 
     @Test
-    public void bytesToConnect() throws UnsupportedEncodingException {
```

Comment:
It would be nice to force on migration so as to avoid big patch :)

Reply:
Got it